File: qe-selenium/qe-selenium-java/src/main/java/com/demos/Activity7.java
package com.demos;

import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity7 {
    public static void main(String[] args) {
        FirefoxDriver driver=new FirefoxDriver();
        driver.get("https://training-support.net/webelements/dynamic-controls/");
        System.out.println(driver.findElement(By.xpath("//*[@id=\"checkbox\"]")).isSelected());
        driver.findElement(By.xpath("//*[@id=\"checkbox\"]")).click();
        System.out.println(driver.findElement(By.xpath("//*[@id=\"checkbox\"]")).isSelected());
        driver.quit();

    }
}

File: qe-selenium/qe-selenium-python/Activity7.py
from selenium import webdriver

from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls/")
    print(driver.title)
    print(driver.find_element(By.XPATH,"//*[@id=\"textInput\"]").is_enabled())
    driver.find_element(By.XPATH,"//*[@id=\"textInputButton\"]").click()
    print(driver.find_element(By.XPATH,"//*[@id=\"textInput\"]").is_enabled())
    driver.quit